apps/web/src/pages/upload.tsx

```diff
@@ -0,0 +1,217 @@
+import { Flex, Text, Heading, Button, Box } from '@chakra-ui/react';
+import { FormTemButtons } from '@web/components/createFormTemplate/FormTemButtons';
+import { SideCreateForm } from '@web/components/createFormTemplate/SideCreateForm';
+import { CloseIcon, PDFIcon, UploadIcon } from '@web/static/icons';
+import { useRouter } from 'next/router';
+import { ChangeEvent, MouseEvent, useCallback, useState } from 'react';
+import { useDropzone } from 'react-dropzone';
+
+export default function Upload() {
+  const router = useRouter();
+  const [pdf, setPdf] = useState<string | ArrayBuffer | null>(null);
```

Comment:
You should be able to pull from main (`git pull origin main`) and use the `useBlob` hook's `localBlobData` state and `uploadLocalBlob` and `clearLocalBlob` functions to do operations regarding storing PDF data in state.